### PR TITLE
Used get_template_directory_uri() instead of get_stylesheet_directory_uri()

### PR DESCRIPTION
**Pull request by shail-mehta**
Poetry & Blue Note both are main theme 

-  We need to add get_template_directory_uri() instead of get_stylesheet_directory_uri(). 
- Also removed Image ids & classes in images

**For Ex:** 
- "id":15
- class="wp-image-15"

### CODE REVIEW

File: blue-note/patterns/media-text.php [hunk 1]
```diff
@@ -6,8 +6,8 @@
  * Block Types: core/media-text
  */
 ?>
-<!-- wp:media-text {"align":"wide","mediaId":146,"mediaLink":"<?php echo esc_url( get_stylesheet_directory_uri() ); ?>/assets/images/John_Coltrane.png","mediaType":"image","mediaWidth":41,"verticalAlignment":"top","imageFill":true,"style":{"spacing":{"padding":{"right":"0","left":"0"}}}} -->
```
Comment (MaggieCabrera) [inline blue-note/patterns/media-text.php:9]: This pattern is showing up broken on the site editor, but the code changes make sense to me, there must be something else

**Review by MaggieCabrera — CHANGES_REQUESTED** [review]
Can you remove the changes to Poetry? They are already included in https://github.com/WordPress/community-themes/pull/152.

If you do that and then change the version number and update the changelog for Blue Note I'll upload the theme with the changes after the merge. Thank you for this!

File: blue-note/readme.txt [hunk 2]
```diff
@@ -11,6 +11,9 @@ Blue Note is an elegant and fun block theme inspired by jazz and the record labe
 
 == Changelog ==
 
+= 1.0.2 =
+* Replaced get_stylesheet_directory_uri() with get_template_directory_uri()
```
Comment (MaggieCabrera) [inline blue-note/readme.txt]:
```suggestion
* Replaced get_stylesheet_directory_uri() with get_template_directory_uri() to ensure child themes work correctly
```

File: blue-note/patterns/media-text.php (hunk 1 above)
Comment (MaggieCabrera) [reply blue-note/patterns/media-text.php:9]: I tested and the problem is with removing the id of the media-text block. Maybe we should open a Gutenberg issue to see why the block markup fails without them. Let's return them so the pattern goes back to working

Comment (shail-mehta) [reply blue-note/patterns/media-text.php:9]: Yeah I just checked with id  and it works fine.

I will revert this pattern's id and class related changes.

Comment (shail-mehta) [reply blue-note/patterns/media-text.php:9]: I have added Image id & class in this pattern & now its working fine for me.

Comment (MaggieCabrera) [reply blue-note/patterns/media-text.php:9]: Fantastic, great job!

**Review by MaggieCabrera — APPROVED** [review]

**Pull request merged**